readme.md

```diff
@@ -26,15 +29,35 @@ tempy.file({name: 'unicorn.png'});
 
 tempy.directory();
 //=> '/private/var/folders/3x/jf5977fn79jbglr7rk0tq4d00000gn/T/2f3d094aec2cb1b93bb0f4cffce5ebd6'
+
+tempy.clean();
```

Comment:
This looks problematic as `tempy` can be used in many places in the process, by other modules even, and they might not expect the temp files to be cleaned underneath their feet.